experiment/secret-sync-controller/go.mod

```diff
@@ -0,0 +1,11 @@
+module b01901143.git/secret-sync
+
+go 1.14
```

Comment:
We use go1.13 in this repo, it would probably be better to follow that

Reply:
switched to 
- go 1.13
- k8s.io/api v0.17.3
- k8s.io/apimachinery v0.17.3
- k8s.io/client-go v0.17.3